### PR TITLE
[SPARK-26044][WEBUI]Aggregated Metrics table sort based on executor ID

### PR DESCRIPTION
**Pull request by shahidki31**
## What changes were proposed in this pull request?
Aggregated Metrics table in the stage page is not sorted based on the executorID properly. Because executorID is string and also the logs of the executors are in the same column. 
In this PR, I created a new column for executor logs.

## How was this patch tested?
Before patch:
screenshot from 2018-11-14 02-05-12

After patch:
screenshot from 2018-11-14 02-05-29

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala [hunk 1]
```diff
@@ -92,16 +93,7 @@ private[ui] class ExecutorTable(stage: StageData, store: AppStatusStore) {
...
-          <div style="float: left">{k}</div>
-          <div style="float: right">
-          {
-            executor.map(_.executorLogs).getOrElse(Map.empty).map {
-              case (logName, logUrl) => <div><a href={logUrl}>{logName}</a></div>
-            }
-          }
-          </div>
-        </td>
+        <td>{if (k == "driver") k else k.toInt} </td>
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala]: I'm curious, why does .toInt matter here? it ends up the same in the HTML, no?

File: core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala [hunk 2]
```diff
@@ -70,6 +70,7 @@ private[ui] class ExecutorTable(stage: StageData, store: AppStatusStore) {
           Blacklisted
           </span>
         </th>
+        <th>Logs</th>
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala]: What about putting the logs link after the ID column, to stay more consistent? or is it that the Logs column is last in other tables?

File: core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala (hunk 1 above)
Comment (shahidki31) [reply core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala]: Thanks. I have updated.

File: core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala (hunk 2 above)
Comment (shahidki31) [reply core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala]: In other tables, like Executors tables, Logs column is in the last. That is why I followed the same here.
screenshot from 2018-11-14 23-29-43

File: core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala [hunk 3]
```diff
@@ -92,16 +93,7 @@ private[ui] class ExecutorTable(stage: StageData, store: AppStatusStore) {
...
-          <div style="float: left">{k}</div>
-          <div style="float: right">
-          {
-            executor.map(_.executorLogs).getOrElse(Map.empty).map {
-              case (logName, logUrl) => <div><a href={logUrl}>{logName}</a></div>
-            }
-          }
-          </div>
-        </td>
+        <td>k</td>
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala]: Does this need to be {k}?

Comment (shahidki31) [reply core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala]: Yes, I updated. Thanks

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Comment by shahidki31** [comment]
Thank you @srowen

**Pull request closed without merging**